### PR TITLE
AWS compress functions

### PR DESCRIPTION
**Pull request by byapparov**

### CODE REVIEW

File: R/aws-s3.R [hunk 1]
```diff
@@ -48,22 +48,76 @@ s3PutFile <- function (df, file.name, path="",
...
+
+#' Saves file as rds to S3
+#'
+#' @inherit s3GetFile
+s3GetFile.rds <- function(path,
+                          bucket = Sys.getenv("AWS_S3_BUCKET"),
+                          root = Sys.getenv("AWS_S3_ROOT")) {
+
+  full.path <- paste0(root, path)
+  s3readRDS(dt, bucket = bucket, object = full.path)
```
Comment (lmavridis) [inline R/aws-s3.R]: Keep consistent with your returns, if we know want to use `invisible` for data.frames/data.tables we should be explicitly use it everywhere

File: R/aws-s3.R [hunk 2]
```diff
@@ -48,22 +48,76 @@ s3PutFile <- function (df, file.name, path="",
...
+
+  tmp.file <- tempfile(fileext = ".gz")
+  on.exit(unlink(tmp.file))
+  gz.connection <- gzfile(tmp.file, "w")
+  write.csv(dt, file = gz.connection, row.names = F, fileEncoding = "UTF-8")
+  close(gz.connection)
+
+  full.path <- paste0(root, path)
+  put_object(file = tmp.file, object = full.path, bucket = bucket)
+
```
Comment (lmavridis) [inline R/aws-s3.R]: Consistent with either an empty line or not

File: R/aws-s3.R (hunk 1 above)
Comment (byapparov) [reply R/aws-s3.R]: that is just a wrapper

File: R/aws-s3.R (hunk 2 above)
Comment (byapparov) [reply R/aws-s3.R]: done )

**Pull request merged**